ui/src/shared/containers/GetFlags.tsx

```diff
@@ -0,0 +1,53 @@
+// Libraries
+import React, {useEffect, FunctionComponent} from 'react'
+import {connect} from 'react-redux'
+
+// Components
+import {SpinnerContainer, TechnoSpinner} from '@influxdata/clockface'
+
+// Types
+import {RemoteDataState, AppState} from 'src/types'
+
+// Actions
+import {getFlags as getFlagsAction} from 'src/shared/actions/flags'
+
+interface PassedInProps {
+  children: React.ReactElement<any>
```

Comment:
love being explicit here, but do we need this?

Reply:
:copy: :pasta: